src/components/layouts/footer/Footer.scss

```diff
@@ -0,0 +1,29 @@
+@import "../../../styles/variables";
+
+.app-footer {
+  height: auto;
+  bottom: 0;
+  position: fixed;
```

Comment:
Since we have fixed footer and header, let's also define some main area (it will be empty for now) so that we make sure that it is visible and not overflowed with fixed elements (you might need margins on it or paddings on the wrapping div).

Reply:
Added `<main />` tag with appropriate style